### DISCUSSION TITLE
Combine Docker containers

### DISCUSSION THREAD

**Discussion by fdw** [Freika/dawarich#234] [Ideas]
Seeing as the code for both necessary Docker containers (app and sidekiq) is already in the same image, would it be possible to have a combined entrypoint so that only one container is necessary?

**Comment by Freika** [comment]
I don't see the benefit of it, at least at the moment.

**Reply by fdw** [reply]
I'd find it easier to set up with only one container running. And I don't see the advantage of having more than one container. Can the sidekiq one started more than once to scale horizontally?

**Reply by Freika** [reply]
We already had a discussion on this very topic in Discord. There are no plans making single unified docker container/image for Dawarich as of now.
